resources/keywords/suite_and_test_teardown.resource

```diff
@@ -0,0 +1,47 @@
+*** Settings ***
+Library     SeleniumLibrary
+Library     BuiltIn
+Library     String
+Library     OpenShiftLibrary
+Library     ../../utils/scripts/dbaas_utils.py
+Library     ../../utils/scripts/util.py
+Library    OpenShiftCLI
+Resource    ../object_repo/login_obj.resource
+Resource    login.resource
+
+
+*** Keywords ***
+Tear Down The Test Suite
+    Log    Test Suite Test Down
+    Login To OpenShift CLI
+    Remove Existing DBaaSNamespaces
+    Remove Existing DBaaSProviderAccounts
+
+Tear Down The Test
+    Remove Existing DBaaSInstances
+    Logout Of OpenShift CLI
+    Close Browser
+
+Remove Existing DBaaSInstances
+    IF  '${instanceName}' != '${EMPTY}'
+       ${cmd_instances}    Set Variable    oc delete -A DBaaSInstance/${instanceName} -n ${newProject} --wait=true
+       ${stdout} =    Execute Command    ${cmd_instances}    True
+    ELSE
+       Log     No DB Instance ${instanceName} found
+    END
+
+Remove Existing DBaaSNamespaces
+    IF  '${newProject}' != '${EMPTY}'
+       ${cmd_namespaces}   Set Variable    oc delete -A Project/${newProject} --wait=true
```

Comment:
Deletion of namespace sometimes get stuck, the basic reason behind this is - some of the resources within the namespace are yet to be removed, and hence the spec finalizer gets stuck waiting for those resources to be removed first.
so to avoid any kind of huge waiting time, we should also include --timeout=Xs with --wait=true